packages/peregrine/lib/talons/Wishlist/GalleryButton/useGalleryButton.ee.js

```diff
@@ -0,0 +1,95 @@
+import { useCallback, useMemo, useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { useIntl } from 'react-intl';
+
+import { useUserContext } from '../../../context/user';
+import operations from './galleryButton.gql';
+import { useGalleryButton as useGalleryButtonCE } from './useGalleryButton.ce';
+
+export const useGalleryButton = props => {
+    const { item, storeConfig } = props;
+
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    const [successToastName, setSuccessToastName] = useState();
+    const [{ isSignedIn }] = useUserContext();
+    const { formatMessage } = useIntl();
+
+    const ceTalonProps = useGalleryButtonCE(props);
```

Comment:
🤔

Reply:
Ha, I warned you 😂  wanted to chat with the team today about this. I know this violates our rule, but would love to discuss alternatives to this that don't duplicate a bunch of code.